src/controllers/userController.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-else-return */
+/* eslint-disable keyword-spacing */
+/* eslint-disable no-unused-vars */
+/* eslint-disable require-jsdoc */
+import sgMail from '@sendgrid/mail';
+import jwt from 'jsonwebtoken';
+import bcrypt from 'bcrypt';
+import Util from '../utils/Utils';
+import Mail from '../utils/Mail';
+import userService from '../services/UserServices';
+import AuthenticationHelper from '../utils/AuthHelper';
+
+const util = new Util();
+const mail = new Mail();
+const { jwtSignReset, jwtVerify } = AuthenticationHelper;
+class UserController {
```

Comment:
There needs to be spacing between line 15 and 16. Make sure you run the linter before committing your work.